.travis.yml

```diff
@@ -28,9 +28,15 @@ env:
     - DD_CASHER_DIR=/tmp/casher
   matrix:
     - TRAVIS_FLAVOR=default
-    - TRAVIS_FLAVOR=core_integration
     - TRAVIS_FLAVOR=checks_mock
 
+matrix:
+  include:
```

Comment:
Yes but actually there are some tests in the `default` flavor that require IPv6 support, which can be only done if the tests are run in the container based environment (`sudo: false`) so I have to separate them.